strongbox-data-service/pom.xml

```diff
@@ -276,7 +276,12 @@
             <groupId>org.reflections</groupId>
             <artifactId>reflections</artifactId>
         </dependency>
-                
+
+        <dependency>            
+            <groupId>org.webjars</groupId>
+            <artifactId>orientdb-studio</artifactId>
```

Comment:
Could you please clarify how [^this^](https://github.com/webjars/orientdb-studio) differs from [OrientDB's project](https://github.com/orientechnologies/orientdb-studio)?

Reply:
`org.webjars:orientdb-studio` is just [webjars.org](https://www.webjars.org/) packaged **OreitntDB Studio**.  Unfortunately I was not able to make it work using original studio distribution from  `'com.orientechnologies:orientdb-studio:jar:3.0.1'` so I used it from **webjars**.